showcase/tests/integration/components/hds/app-header/index-test.js

```diff
@@ -157,13 +157,13 @@ module('Integration | Component | hds/app-header/index', function (hooks) {
 
   // A11Y Refocus
 
-  test('it renders the `a11y-refocus` elements by default with a default skip link href value of "#main', async function (assert) {
+  test('it renders the `a11y-refocus` elements by default with a default skip link href value of "#hds-main', async function (assert) {
```

Comment:
question: should we add a test for a custom `id`, to make sure that potential future changes will not regress this?

Reply:
I will add a test for `AppFrame::Main` passing a custom id value as I think that's a good idea.

(The AppHeader tests already include a custom value for the skip link target using the `@a11yRefocusSkipTo` argument.)

Reply:
Added tests for the default id value and a custom value.